auth/README.md

```diff
@@ -35,6 +35,11 @@ You can then run a given `ClassName` via:
     mvn exec:java -Dexec.mainClass=com.google.cloud.auth.samples.AuthExample
         -Dexec.args="compute"
 
+### Authenticating using an api key
+
+    mvn exec:java -Dexec.mainClass=com.google.cloud.auth.samples.AuthExample
+        -Dexec.args="apikey [api key]"
```

Comment:
Can you add a reference where one can get an API key?

Also would you make it clear that "[api key]" should be replaced with the value of an API key?